octobot_trading/exchanges/util/parser/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .trades_parser import *
```

Comment:
Can you use the same import method as elsewhere please?

Reply:
(from x import y is hiding cyclic imports)

Reply:
is fixed